presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1542,7 +1555,7 @@ else if (hasAggregates(node)) {
                     groupByExpression = groupingColumn;
                 }
 
-                Analyzer.verifyNoAggregatesOrWindowFunctions(metadata.getFunctionRegistry(), groupByExpression, "GROUP BY clause");
+                Analyzer.verifyNoAggregateWindowOrGroupingFunctions(metadata.getFunctionRegistry(), groupByExpression, "GROUP BY clause");
```

Comment:
This is sufficient to report an error for a query `select grouping(a) from (values 1) t(a) group by 1` but the error location is not accurate: the error location is `grouping(a)` instead of `group by 1`.